extra/resources/ClusterMon

```diff
@@ -244,6 +247,8 @@ fi
 
 OCF_RESKEY_update=`expr $OCF_RESKEY_update / 1000`
 
+CMON_CMD="${HA_SBIN_DIR}/crm_mon -p $OCF_RESKEY_pidfile -d -i $OCF_RESKEY_update $OCF_RESKEY_extra_options -h $OCF_RESKEY_htmlfile"
```

Comment:
This will work as long as pidfile and htmlfile don't contain spaces, but that's fine with me -- it would cause problems elsewhere anyway.